assignments/callbacks.js

```diff
@@ -24,23 +24,31 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  const length = arr.length;
+  return cb(length);
 }
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  const i = arr.length-1;
+  return cb(arr[i]);
```

Comment:
Nicely done! If you wanted to use one less line of code, you could write `return cb(arr[arr.length-1])` and remove line 32.